tasks/db_deploy/migrate_db.py

```diff
@@ -157,3 +162,32 @@ def migrate_versions_1_to_2(config: Dict[str, str], is_latest_version: bool) ->
 
         # Commit if there is no issues
         connection.commit()
+
+
+def migrate_versions_2_to_3(config: Dict[str, str], is_latest_version: bool) -> None:
+    """
+    Performs the migration of the ORCA schema from version 1 to version 2 of
+    the ORCA schema.
+
+    Args:
+        config (Dict): Connection information for the database.
+        is_latest_version (bool): Flag to determine if version 2 is the latest schema version.
+    """
+    # Get the admin engine to the app database
+    admin_app_connection = get_admin_connection(config, config["database"])
+
+    # Create all of the new objects, users, roles, etc.
+    with admin_app_connection.connect() as connection:
+        # Add column multipart_chunksize_mb to orca_files
```

Comment:
You will want to call the function that sets the path first so the table is installed in the orca schema.

```
# Change to DBO role and set search path 
logger.debug("Changing to the dbo role to modify objects ...")
connection.execute(text("SET ROLE orca_dbo;"))
logger.debug("Setting search path to the ORCA schema to modify objects ...")
connection.execute(text("SET search_path TO orca, public;"))
```

Reply:
Fixed